samples/recognize.v1p1beta1.js

```diff
@@ -245,6 +245,65 @@ function syncRecognizeWithMetaData(
   // [END speech_transcribe_file_with_metadata]
 }
 
+function syncRecognizeWithEnhancedModel(
+  filename,
+  encoding,
+  sampleRateHertz,
+  languageCode
+) {
+  // [START speech_transcribe_file_with_enhanced_model]
+  // Imports the Google Cloud client library
+  const fs = require('fs');
+
+  // Imports the Google Cloud client library for Beta API
+  /**
+   * TODO(developer): Update client library import to use new
+   * version of API when desired features become available
+   */
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const filename = 'Local path to audio file, e.g. /path/to/audio.raw';
+  // const encoding = 'Encoding of the audio file, e.g. LINEAR16';
+  // const sampleRateHertz = 16000;
+  // const languageCode = 'BCP-47 language code, e.g. en-US';
+
+  const config = {
+    encoding: encoding,
+    languageCode: languageCode,
+    useEnhanced: true,
+    model: 'phone_call',
```

Comment:
*As a reader, I wonder:*
 - What are the other available models?  
 - What can I use besides `phone_call`?  
 - Why isn't the `model` parameter overridable via CLI?

Reading this sample makes me think to myself:

> *"Huh, the sample which was written to demonstrate how to choose an enhanced model has detailed descriptions of `filename`, `encoding`, `sampleRateHertz`, and `languageCode`. How come it does not include detailed descriptions for `model` and `useEnhanced`? These seem to be the relevant fields for this sample. What's the deal?"*

Reply:
Fair point. I also wondered about other options besides `phone_call` and also wondered what I would get extra by using `phone_call`. It should definitely be documented somewhere. The question is whether it should be in the sample code or the Doc pages. If it should be in the sample code, we also need to modify other sample codes accordingly.

Reply:
How about just allowing the user to pass in `model`? The same way the `speech_transcribe_model_selection` sample does (in this same file)

I think that's more than sufficient 😄

Reply:
^ I'm LGTM after adding `model` as a parameter

Take a look at `speech_transcribe_model_selection`. In addition to adding `model` to the function as a parameter (and adding a commented out variable placeholder for it), you also need to tell the command-line to support the model param ---> see example here:
https://github.com/googleapis/nodejs-speech/blob/687d52701641f3c517a799fae73103d85955e16c/samples/recognize.v1p1beta1.js#L322-L334